OpenRA.Mods.Common/Widgets/Logic/SettingsLogic.cs

```diff
@@ -394,7 +394,7 @@ Action InitInputPanel(Widget panel)
 					{ "CycleBaseKey", "Jump to base" },
 					{ "ToLastEventKey", "Jump to last radar event" },
 					{ "ToSelectionKey", "Jump to selection" },
-					{ "SelectAllUnitsKey", "Select all units on screen" },
+					{ "SelectAllUnitsKey", "Select all military units" },
```

Comment:
I don't have any particular opinions on this - but I'm commenting here to draw attention for other reviewers. :)